app/views/past_foreign_secretaries/index.html.erb

```diff
@@ -19,152 +19,112 @@
     <ol>
       <li class="person person-excerpt">
         <div class="inner">
-          <div class="image-holder govuk-!-margin-bottom-1">
-            <a href="/government/history/past-foreign-secretaries/edward-wood" class="aspect-ratio-3:2" tabindex="-1" aria-hidden="true">
-              <%= image_tag 'history/past-foreign-secretaries/viscount-halifax.jpg', alt: 'Edward Frederick Lindley Wood, Viscount Halifax', loading: "lazy" %>
-            </a>
-          </div>
-          <p class="govuk-heading-s govuk-!-margin-bottom-1">
-            <a href="/government/history/past-foreign-secretaries/edward-wood" class="govuk-link">
-              Edward Frederick Lindley Wood, Viscount Halifax
-            </a>
-          </p>
-          <p class="govuk-body-s govuk-!-margin-bottom-0 term">1938 to 1940</p>
+          <%= render "govuk_publishing_components/components/image_card", {
```

Comment:
Are we ok with this change on mobile? 🤔  

![Screenshot 2021-03-04 at 10 56 36](https://user-images.githubusercontent.com/71266765/109953697-51acee80-7cd8-11eb-97fb-ac9c01bd4caa.png)

Reply:
Good spot! I personally think that's fine... It's the standard behaviour coming from the image card. [Live usage](https://www.gov.uk/government/organisations/attorney-generals-office). I'm happy to get a design eye on this if we want?

Reply:
They do seem a bit small, esp on like a 4-inch device.  We don't use images much nor does the image card support the previous layout.

Design comment might be good thought just in case.

However, if you're happy, I'm good -  👍🏼   the default behaviour angle I get.

Reply:
I'll run this past Mia just for safety's sake.

Reply:
We've agreed that we're gonna merge this for now and I might drop the component past Mia at a later date.